modules/nextflow/src/main/groovy/nextflow/cli/CmdConfig.groovy

```diff
@@ -113,6 +116,10 @@ class CmdConfig extends CmdBase {
 
         final config = builder.buildConfigObject()
 
+        // -- validate config options
+        new ConfigValidator().validate(config)
```

Comment:
Nit, what about adding a shortcut static `validate` (likely it would require to rename current `validate` to something else e.g. `validateConfig`

Reply:
meh. it's not really needed